### PR TITLE
Added sr-Cyrl-RS localization

### PR DESCRIPTION
**Pull request by gwynjudd**

### CODE REVIEW

File: tests/languages/locale-codes.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ exports['locale-codes'] = {
         test.expect(keys.length);
         keys.forEach(function(key) {
             var lang = cultures[key];
-            test.strictEqual(true, /^[a-z]{2,3}(?:-[A-Z]{2,4}(?:-[a-zA-Z]{4})?)?$/g.test(lang.langLocaleCode),
+            test.strictEqual(true, /^[a-z]{2,3}(?:-[a-zA-Z]{2,4}(?:-[a-zA-Z]{2,4})?)?$/g.test(lang.langLocaleCode),
```
Comment (BenjaminVanRyseghem) [inline tests/languages/locale-codes.js]: why this change here?

**Comment by gwynjudd** [comment]
Because the Cyrl bit doesn't match [A-Z]{2,4} you have to allow for lower case letters

On 30 Mar. 2017 10:00 pm, Benjamin Van Ryseghem <notifications@github.com> wrote:

@BenjaminVanRyseghem commented on this pull request.

________________________________

In tests/languages/locale-codes.js<https://github.com/foretagsplatsen/numbro/pull/267#discussion_r108874291>:

> @@ -52,7 +52,7 @@ exports['locale-codes'] = {
         test.expect(keys.length);
         keys.forEach(function(key) {
             var lang = cultures[key];
-            test.strictEqual(true, /^[a-z]{2,3}(?:-[A-Z]{2,4}(?:-[a-zA-Z]{4})?)?$/g.test(lang.langLocaleCode),
+            test.strictEqual(true, /^[a-z]{2,3}(?:-[a-zA-Z]{2,4}(?:-[a-zA-Z]{2,4})?)?$/g.test(lang.langLocaleCode),


why this change here?
